systemtest/src/test/java/io/strimzi/systemtest/security/NetworkPoliciesST.java

```diff
@@ -114,11 +114,11 @@ void testNetworkPoliciesOnListenersWhenOperatorIsInSameNamespaceAsOperands() {
 
         resourceManager.createResourceWithWait(
             NodePoolsConverter.convertNodePoolsIfNeeded(
-                KafkaNodePoolTemplates.brokerPool(testStorage.getNamespaceName(), testStorage.getBrokerPoolName(), testStorage.getClusterName(), 1).build(),
-                KafkaNodePoolTemplates.controllerPool(testStorage.getNamespaceName(), testStorage.getControllerPoolName(), testStorage.getClusterName(), 1).build()
+                KafkaNodePoolTemplates.brokerPool(testStorage.getNamespaceName(), testStorage.getBrokerPoolName(), testStorage.getClusterName(), 3).build(),
+                KafkaNodePoolTemplates.controllerPool(testStorage.getNamespaceName(), testStorage.getControllerPoolName(), testStorage.getClusterName(), 3).build()
```

Comment:
I know that you are maybe fixing some testcases during your debug, but it should be at least mentioned in the description
Because it's unrelated to the whole idea of the PR - or you can fix it in separate PR, which would make more sense